api/v1beta1/restore_types.go

```diff
@@ -80,6 +82,12 @@ type RestoreSpec struct {
 	//
 	// if not defined, the value is assumed to be CleanupTypeNone - no clean up called
 	CleanupBeforeRestore CleanupType `json:"cleanupBeforeRestore,omitempty"`
+	// +kubebuilder:validation:Optional
+	// Set this to true if you want to keep checking for new backups and restore if updates are available.
+	// If not defined, the value is set to false.
+	// For this option to work, you need to set VeleroResourcesBackupName and VeleroCredentialsBackupName
```

Comment:
I think this should be : For this option to work, you need to set VeleroResourcesBackupName, VeleroCredentialsBackupName and veleroManagedClustersBackupNameto to latest or skip. If you specify a backup name for any of these options, the SyncRestoreWithNewBackups will be ignored.